packages/vira-book/src/element-book/entries/dropdown/vira-dropdown.element.book.ts

```diff
@@ -0,0 +1,206 @@
+import {isTruthy} from '@augment-vir/common';
+import {BookPageControlTypeEnum, defineBookPage, definePageControl} from 'element-book';
+import {CSSResult, html, listen} from 'element-vir';
+import {ViraDropdown, ViraDropdownOption, allIconsByName} from 'vira';
+import {dropdownPage} from './vira-dropdown.book';
+
+const exampleDropdownOptions: ReadonlyArray<Readonly<ViraDropdownOption>> = [
+    {
+        label: 'Option 1',
+        id: 1,
+    },
+    {
+        label: 'Option 2',
+        id: 2,
+    },
+    {
+        label: 'Option 3',
+        id: 3,
+    },
+    {
+        label: 'Really really super duper long option',
+        id: 4,
+    },
+    {
+        label: 'Really really super duper long option',
+        id: 5,
+    },
+    {
+        label: 'Really really super duper long option',
+        id: 6,
+    },
+    {
+        label: 'Really really super duper long option',
+        id: 7,
+    },
+    {
+        label: "Really really super duper long it just keeps going because it's so long option",
+        id: 8,
+    },
+];
+
+const examples: ReadonlyArray<{
+    title: string;
+    inputs?: Partial<typeof ViraDropdown.inputsType>;
+    customStyle?: CSSResult;
+}> = [
+    {
+        title: 'default',
+    },
+    {
+        title: 'disabled',
+        inputs: {
+            isDisabled: true,
+        },
+    },
+    {
+        title: 'multi select',
+        inputs: {
+            isMultiSelect: true,
+        },
+    },
+    {
+        title: 'long selection',
+        inputs: {
+            selected: [8],
+        },
+    },
+    {
+        title: 'with custom template',
+        inputs: {
+            selected: [8],
+            options: [
+                ...exampleDropdownOptions,
+                {
+                    id: 42,
+                    label: 'custom template',
+                    template: html`
+                        <select>
+                            <option selected>NESTED SELECT!!!</option>
+                            <option>this is a terrible idea</option>
+                            <option>pls don't do this</option>
+                        </select>
+                    `,
+                },
+            ],
+        },
+    },
+    {
+        title: 'with disabled item',
+        inputs: {
+            selected: [8],
+            options: [
+                ...exampleDropdownOptions,
+                {
+                    id: 42,
+                    label: 'this is disabled',
+                    disabled: true,
+                },
+            ],
+        },
+    },
+    // {
```

Comment:
This is your personal repo, so I dont know what conventions you have enforced. But I would avoid having commented code.

Reply:
good catch!

Reply:
fixed